test/TestHdf5Image.cc

```diff
@@ -22,29 +22,29 @@ class TestFrame : public Frame {
     FRIEND_TEST(Hdf5ImageTest, ExampleFriendTest);
 };
 
-class Hdf5ImageTest : public ::testing::Test, public ImageGenerator {};
+class Hdf5ImageTest : public ::testing::Test {};
 
 TEST_F(Hdf5ImageTest, BasicLoadingTest) {
-    auto path_string = GeneratedHdf5ImagePath("10 10");
-    std::shared_ptr<carta::FileLoader> loader(carta::FileLoader::GetLoader(path_string));
+    auto path = Hdf5Images() / "10_10_row_column.hdf5";
```

Comment:
The original images used in these tests don't have any NaNs (although it shouldn't make any difference).